internal/provider/incident_workflow_resource.go

```diff
@@ -401,7 +384,7 @@ func (r *IncidentWorkflowResource) buildConditionGroups(groups []client.Expressi
 }
 
 func (r *IncidentWorkflowResource) buildConditions(conditions []client.ConditionV2) []IncidentEngineCondition {
-	var out []IncidentEngineCondition
+	out := []IncidentEngineCondition{}
```

Comment:
so that we always end up with an empty json array , to not hit `required` validation errs ?

Reply:
Yep exactly that.